src/lib/ledger_proof/ledger_proof.ml

```diff
@@ -69,19 +66,8 @@ struct
   let create ~statement ~sok_digest ~proof:_ = (statement, sok_digest)
 end
 
-[%%if
-proof_level = "full"]
-
 include Prod
```

Comment:
is the `Debug` module dead code? still keep it for debugging?

Reply:
I suppose it is, yea. I'll do that in a follow-up